build.sbt

```diff
@@ -163,10 +79,10 @@ lazy val npmBetaReleaseTagMaybe =
 
 lazy val typescriptClasses = (project in file("ts"))
   .enablePlugins(ScroogeTypescriptGen)
-  .settings(commonSettings)
-  .settings(npmBetaReleaseTagMaybe)
+  .settings(artifactProductionSettings)
+  .settings(npmBetaReleaseTagMaybe)//do we need to remove this?
```

Comment:
Yep, we need to remove `npmBetaReleaseTagMaybe`, along with `betaReleaseType` & `betaReleaseSuffix`, etc!